docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,616 @@ license: |
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the exact percentile value of numeric expression at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td><b>percentile</b>(<i>expression, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>numeric type; double; integral type</td>
+      <td>Percentage array is an array of number between 0 and 1; frequency is a positive integer. Returns the exact percentile value array of numeric expression at the given percentage(s).</td>
+    </tr>        
+    <tr>
+      <td><b>{percentile_approx | percentile_approx}</b>(<i>expression, percentage [, frequency]</i>)</td>
+      <td>numeric, date, timestamp; double; integral</td>
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the approximate percentile value of numeric expression at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td><b>{percentile_approx | percentile_approx}</b>(<i>expression, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
+      <td>numeric, date, timestamp; double; integral</td>
+      <td>Percentage is a number between 0 and 1; Frequency is a positive integer. Returns the approximate percentile value of numeric expression at the given percentage.</td>
+    </tr>             
+    <tr>
+      <td><b>skewness</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the skewness value calculated from values of a group.</td>
+    </tr>    
+    <tr>
+      <td><b>{stddev_samp | stddev | std}</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the sample standard deviation calculated from values of a group.</td>
+    </tr>  
+    <tr>
+      <td><b>stddev_pop</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the population standard deviation calculated from values of a group.</td>
+    </tr>
+    <tr>
+      <td><b>sum</b>(<i>expression</i>)</td>
+      <td>numeric</td>
+      <td>Returns the sum calculated from values of a group.</td>
+    </tr>       
+    <tr>
+      <td><b>{variance | var_samp}</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the sample variance calculated from values of a group.</td>
+    </tr>    
+    <tr>
+      <td><b>var_pop</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the population variance calculated from values of a group.</td>
+    </tr>        
+  </tbody>
+</table>
+
+### Examples
+{% highlight sql %}
+--base table 
+SELECT * FROM buildin_agg;
++----+----+----+-----+----+
+|  c1|  c2|  c3|   c4|  c5|
++----+----+----+-----+----+
+|   2|   3|agg4| true|true|
+|   1|   2|agg3|false|true|
+|   1|   1|agg1|false|true|
+|   4|   3|agg6|false|true|
+|   3|   3|agg5| true|true|
+|   1|   2|agg2|false|true|
+|   5|null|agg8|false|true|
+|null|   4|agg7|false|true|
++----+----+----+-----+----+
+
+-- ANY, SOME and BOOL_OR
```

Comment:
nit: the lowercase is better along with the function list above? `-- any, some and bool_or`

Reply:
ok